concourse/tasks/gpdb_packaging.yml

```
@@ -1,15 +0,0 @@
-platform: linux
```

If this stays out, the packaging jobs need to be ripped out of the pipelines. If we don't want to do that, we'll need to put this file back before we can merge this PR.

I lean towards leaving the packaging job in the master pipeline. It will be better to see a failure in packaging there, rather than in the release pipeline. Fail fast!